### PR TITLE
[wallet/desktop] releasing: v1.0.13

### PR DESCRIPTION
**Pull request by yilmazbahadir**

### CODE REVIEW

File: src/config/NetworkConfig.ts [hunk 1]
```diff
@@ -74,7 +74,7 @@ export const defaultTestnetNetworkConfig: NetworkConfig = {
         defaultDynamicFeeMultiplier: 100,
         epochAdjustment: 1637848847,
         totalChainImportance: 7842928625000000,
-        generationHash: '7FCCD304802016BEBBCD342A332F91FF1F3BB5E902988B352697BE245F48E836',
+        generationHash: 'A7D2BA9249F3F3874DC93E5790C4E4D4C934B0EDC3DE02D9DC593486688E75F9',
```
Comment (cryptoBeliever) [inline src/config/NetworkConfig.ts:77]: It should be 49D6E1CE276A85B70EAFE52349AACCA389302E7A9754BCF1221E79494FC665A4

Comment (cryptoBeliever) [reply src/config/NetworkConfig.ts:77]: `currencyMosaicId` and `harvestingMosaicId` should be 72C0212E67A08BCE

Comment (yilmazbahadir) [reply src/config/NetworkConfig.ts]: ok, updated

File: src/config/NetworkConfig.ts [hunk 2]
```diff
@@ -69,12 +69,12 @@ export const defaultTestnetNetworkConfig: NetworkConfig = {
         maxCosignedAccountsPerAccount: 25,
         maxMessageSize: 1024,
         maxMosaicAtomicUnits: 8999999999000000,
-        currencyMosaicId: '3A8416DB2D53B6C8',
-        harvestingMosaicId: '3A8416DB2D53B6C8',
+        currencyMosaicId: '72C0212E67A08BCE',
+        harvestingMosaicId: '72C0212E67A08BCE',
         defaultDynamicFeeMultiplier: 100,
         epochAdjustment: 1637848847,
```
Comment (AnthonyLaw) [inline src/config/NetworkConfig.ts:75]: should the epochAdjustment change, it looks different from the network.

Comment (yilmazbahadir) [reply src/config/NetworkConfig.ts]: updated

**Review by AnthonyLaw — APPROVED** [review]
Look good 👍🏼

**Review by AnthonyLaw — APPROVED** [review]

File: src/config/NetworkConfig.ts (hunk 2 above)
Comment (cryptoBeliever) [reply src/config/NetworkConfig.ts]: good point @AnthonyLaw :+1:

**Review by cryptoBeliever — APPROVED** [review]

**Review by gimre-xymcity — APPROVED** [review]

**Pull request merged**